target/Extensions/UINavigationControllerExtension.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  UINavigationControllerExtension.swift
+//  target
+//
+//  Created by Cecilia Domingo on 5/28/20.
+//  Copyright © 2020 TopTier labs. All rights reserved.
+//
+
+import UIKit
+
+extension UINavigationController {
+  
+  func setupNavigationBar() {
+    setNavigationBarHidden(false, animated: true)
+    navigationBar.tintColor = .black
+    navigationBar.barTintColor = .white
+    navigationBar.titleTextAttributes = [NSAttributedString.Key.foregroundColor: UIColor.black, NSAttributedString.Key.font: UIFont(name: "OpenSans-SemiBold", size: 13) ?? UIFont.systemFont(ofSize: 13)]
```

Comment:
Please split this line so it's easier to read. Also, I think you can avoid the `NSAttributedString.Key.` prefix.
```suggestion
    navigationBar.titleTextAttributes = [
      .foregroundColor: UIColor.black,
      .font: UIFont(name: "OpenSans-SemiBold", size: 13) ?? UIFont.systemFont(ofSize: 13)
    ]
```

Another thing I'd do is take advantage of `FontExtension`, which defaults the font for you. You can add some other helpers there to set a fallback weight and define specific functions for each font so you don't have to type the font name every time.